packages/yosqlite/yosqlite.0.1/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Simple Sqlite3. Yojson+Sqlite3 amalgamation"
+description: "Convenience functions for working with Sqlite3 library"
+maintainer: ["Szymon Wygnański"]
```

Comment:
Could you provide some way of contacting you here please? An email address is our usual way.